opcua/common/ua_utils.py

```diff
@@ -238,3 +238,36 @@ def get_variable_basetype(server, datatype_id):
     if not dtype_supers_nodeids:
         raise ua.UaError("Datatype must be a subtype of builtin types %s" % datatype_id.nodeid)
     return dtype_supers_nodeids[0]
+
+def get_nodes_of_namespace(server, namespaces=[]):
+    """
+    Get the nodes of one or more namespaces .      
+    Args:
+        server: opc ua server to use
+        namespaces: list of string uri or int indexes of the namespace to export
+    Returns:
+        List of nodes that are part of the provided namespaces
+    """
+    ns_available = server.get_namespace_array()
+    ns_count = len(ns_available)
+
+    if not namespaces:
+        namespaces = ns_available[1:]
+
+    if type(namespaces) is not list and type(namespaces) in [str, int]:
+        namespaces = [namespaces]
+
+    namespace_indexes = []
+    for namespace in namespaces:
+        if type(namespace) is str:
+            namespace = server.get_namespace_index(namespace)
+        if namespace >= ns_count:
+            raise ua.UaError("Namespace index '%d' is invalid" % namespace)
+        namespace_indexes.append(namespace)
```

Comment:
This would lead to:
```python
 namespace_indexes = [ n if isinstance(n, int) else ns_available.index(n) for n in namespaces ]
 if max(namespace_indexes) >= len(ns_available):
    raise ua.UaError("Namespace index '%d' is invalid" % max(namespace_indexes))
```
Not sure if this makes the code more readable and adding comment to explain it feels unnatural...
Also the export is not really time critical.
But in the end it is your party, so I will make the change ;-)